index.php

```diff
@@ -9,34 +9,80 @@
   <!-- main posts loop -->
   <section id="posts" class="container">
 <?php
-if( have_posts() ) {
-  while( have_posts() ) {
-    the_post();
+  $categories = get_terms('category', array('orderby' => 'id', 'hide_empty' => false));
+
+  if ($categories) {
+    $i = 1;
+    foreach($categories as $category) {
```

Comment:
its normal foreach syntax
```
$foo = array( 'a', 'b', 'c');

foreach($foo as $key => $value) {
  echo 'Key is: ' . $key . '. Value is: ' . $value . '\n';
}

> Key is: 0. Value is: a
> Key is: 1. Value is: b
> Key is: 2. Value is: c
```
you could name `$key` as `$index` or whatever suits your needs, same for `$value`.
```
$chiefs = array(
 'CEO' => 'Cas',
 'CTO' => 'Pancho',
 'CMO' => 'Miguelito',
);

foreach($foo as $position => $name) {
  echo $name . ' is the ' .  $position . ' of IGV';
}
> Cas is the CEO of IGV
> Pancho is the CTO of IGV
> Miguelito is the CMO of IGV
```